pkg/test/echo/common/util.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	ConnectionTimeout     = time.Second * 2
+	ConnectionTimeout     = 5 * time.Second
```

Comment:
Please don't change this, it impacts multiple places. I am ok with doing this change to just the single call site and *temporarily* until we resolve these DNS/wasm issues, but we need to create a release blocker issue to revert this change. It is a clear bug that we cannot send traffic for a full 2 seconds, do not want to just hide it

Reply:
Ah, I thought it is the environment issue.

Reply:
It does still seem to he failing so I'm not sure why, it's a clear regression. I am fine with changing this if you change it just at the init call site but this variable is used in multiple places and we only want to change the init

Reply:
The case here is: 
 grpc connect to a pod via kube forwarder. No dns resolution at all.

maybe i should change this connection timeout to `pkg/test/echo/client/client.go`